.github/workflows/formulaCI.yml

```diff
@@ -0,0 +1,78 @@
+name: Formula CI
+# this workflow will
+# - run formula validation step - test/validate-formula.sh
```

Comment:
Just as a comment, this is wrong. It should be `ci/validate-formula.sh`

Reply:
i updated the salt version from latest to 3000 and corrected the comment